tests/fixtures/transform_custom.py

```diff
@@ -29,3 +36,55 @@ def transform(X, transformer, y=None):
         )
     else:
         return pd.DataFrame(transformed)
+
```

Comment:
can we remove the top-level functions if we have this?

Reply:
same question for the other fixtures.  so in this one I'm asking if we can delete L#18-L#38

Reply:
I _think_ this means that you've defined the task twice in a single file and there's a real question of what will happen when you try to run this, and with two things here, that behavior could change at any time.

Reply:
Sorry, I don't understand what `twice` refers to.
If I do:
`drum fit --code-dir task_templates/1_transforms/3_python3_sklearn_transform --input tests/testdata/10k_diabetes_sample.csv --target-type transform --target readmitted --output ~/custom_task`

In the output dir I get:
ls -la  ~/custom_task/
total 44
-rw-rw-r-- 1 yakov.goldberg yakov.goldberg  1762 Jul  9 15:37 create_transform_pipeline.py
-rw-rw-r-- 1 yakov.goldberg yakov.goldberg  1762 Apr 14 23:22 custom.py
-rw-rw-r-- 1 yakov.goldberg yakov.goldberg 22703 Jul 10 10:08 drum_artifact.pkl
-rw-rw-r-- 1 yakov.goldberg yakov.goldberg  3099 Nov 16  2023 model-metadata.yaml
drwxrwxr-x 2 yakov.goldberg yakov.goldberg  4096 Jul 10 10:08 __pycache__
-rw-rw-r-- 1 yakov.goldberg yakov.goldberg   138 Jul 10 10:08 README.md


drum score works when i predict on that folder.
custom.py contains CustomTask

Reply:
The custom task is defined twice in one file.  Once with the old style hooks and again with the new interface class.  That is confusing and I am actually shocked that it works at all.

Reply:
oh,
why shouldn't it work... drum code just loads `CustomTask`

Reply:
Yes, you can remove the old method that isn't used anymore if you are using the interface class.

Reply:
removed old methods